Header header
int64 robotID
geometry_msgs/Point32 location
geometry_msgs/Point32 forwardVec
time lastPosLock
time lastRotLock
